From 819d11507f6637731947836e6308f5966d64cf9d Mon Sep 17 00:00:00 2001
Message-Id: <819d11507f6637731947836e6308f5966d64cf9d.1650795601.git.geliang.tang@suse.com>
From: Colin Ian King <colin.i.king@gmail.com>
Date: Fri, 17 Dec 2021 18:24:00 +0000
Subject: [PATCH] bpf, selftests: Fix spelling mistake "tained" -> "tainted"
Git-commit: 819d11507f6637731947836e6308f5966d64cf9d
Patch-mainline: v5.16
References: bsn#127
Modified-by-SEL: No


There appears to be a spelling mistake in a bpf test message. Fix it.

Signed-off-by: Colin Ian King <colin.i.king@gmail.com>
Signed-off-by: Daniel Borkmann <daniel@iogearbox.net>
Link: https://lore.kernel.org/bpf/20211217182400.39296-1-colin.i.king@gmail.com
Signed-off-by: Geliang Tang <geliang.tang@suse.com>
---
 tools/testing/selftests/bpf/verifier/value_ptr_arith.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/testing/selftests/bpf/verifier/value_ptr_arith.c b/tools/testing/selftests/bpf/verifier/value_ptr_arith.c
index 4d347bc53aa2..359f3e8f8b60 100644
--- a/tools/testing/selftests/bpf/verifier/value_ptr_arith.c
+++ b/tools/testing/selftests/bpf/verifier/value_ptr_arith.c
@@ -1078,7 +1078,7 @@
 	.errstr_unpriv = "R0 pointer -= pointer prohibited",
 },
 {
-	"map access: trying to leak tained dst reg",
+	"map access: trying to leak tainted dst reg",
 	.insns = {
 	BPF_MOV64_IMM(BPF_REG_0, 0),
 	BPF_ST_MEM(BPF_DW, BPF_REG_10, -8, 0),
-- 
2.26.2

